packages/core/bootstrap/test/unit/cache-warmer.test.ts

```diff
@@ -52,6 +52,7 @@ describe('side effect tests', () => {
   }
   const adapterRequest1: AdapterRequest = { data: {}, id: '0' }
   const adapterRequest2: AdapterRequest = { data: { foo: 'bar' }, id: '0' }
+  const adapterRequest3: AdapterRequest = { foo: 'bar', id: '0' }
```

Comment:
What is this new request for?

We expect request data to come in within `data`.

Reply:
So the parent and child warmers have slightly different shapes stored in Redux.  The parent stores it as 

```
{
  origin: {
    base: ...,
    quote: ...
  }
}
```

while the child stores it like this

```
{
   origin: {
      data: {
        base: ...,
        quote: ...
    }
}
```
```

Reply:
I'm not following. Did you mean that flipped around?

It gets used in the code @ 
https://github.com/smartcontractkit/external-adapters-js/pull/744/files/4056e37cbfd55348b25499946bf5e39d6fd97aea#diff-0b96592c3d7aac6f7a84adba3fd48d9219a737782f051a9875244e668fbcb9efR424
which is setting a child state.

Reply:
Should be able to change this now.